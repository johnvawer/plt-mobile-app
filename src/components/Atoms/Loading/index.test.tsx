import React from 'react'
import { render } from '@testing-library/react-native'
import Loading from '.'

describe('<Loading />', () => {
  it('should render the element correctly', () => {
    const { toJSON } = render(<Loading />)

    expect(toJSON()).toMatchSnapshot()
  })
})
