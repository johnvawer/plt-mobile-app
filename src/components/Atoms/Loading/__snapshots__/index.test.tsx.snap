// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> should render the element correctly 1`] = `
<View>
  <Text>
    Loading
  </Text>
</View>
`;
